Dockerfile

```diff
@@ -1,9 +1,9 @@
 # SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
-ARG NODE_VERSION=20.0.0
+ARG NODE_VERSION=20
 
-FROM node:${NODE_VERSION}-alpine
```

Comment:
so not alpine version anymore?

Reply:
I had some trouble there with the arm build, but can see if I get it working somehow in a follow up

Reply:
Follow-up with Alpine would be cool as packages are generally more up-to-date

Reply:
Pushed, seemed to have worked without issues now. Not sure what the problem was before